app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -28,11 +31,13 @@ import '../service/logging.dart';
 @immutable
 class PostsubmitLuciSubscription extends SubscriptionHandler {
   /// Creates an endpoint for listening to LUCI status updates.
-  const PostsubmitLuciSubscription(
-    CacheService cache,
-    Config config, {
+  const PostsubmitLuciSubscription({
+    required CacheService cache,
+    required Config config,
     AuthenticationProvider? authProvider,
     @visibleForTesting this.datastoreProvider = DatastoreService.defaultProvider,
+    required this.luciBuildService,
+    required this.scheduler,
   }) : super(
```

Comment:
Are we using this topic `luci-postsubmit` anywhere now?

Reply:
Yes, it's used for targets that are using the cocoon scheduler (so cocoon, plugins, packages, and the engine's ci.yaml roller)

Reply:
Could you point me to the locations referencing this topic? I am not able to find..

Reply:
For scheduling post-submit builds, we add it here : https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/service/luci_build_service.dart;l=503

Reply:
The topic here is `luci-postsubmit`, where the referenced line uses topic `luci-builds-prod`. Is this intended?

Reply:
There's two parts to pubsub:

1. Topics. This is generic, and requires a subscription
2. Subscription. There can be N of these per topic to support multiple consumers

With that said, BuildBucket publishes the messages to the `luci-postsubmit` topic, and Cocoon has a single subscription of `luci-builds-prod` to get the messages. In the future, someone might add another subscription dedicated for the results processing aspect

Reply:
Ah, that makes sense. Thanks for the explanations.